default.xml

```diff
@@ -37,6 +37,9 @@
   <project path="development" name="android_development" remote="los" />
 
   <project path="device/google/contexthub" name="device/google/contexthub" groups="device,marlin,pdk" remote="aosp" />
+  <project path="device/qcom/sepolicy" name="android_device_qcom_sepolicy" groups="qcom" remote="los" />
+  <project path="device/qcom/common" name="android_device_qcom_common" groups="qcom" remote="los" />
+  <project path="vendor/qcom/opensource/cryptfs_hw" name="android_vendor_qcom_opensource_cryptfs_hw" groups="qcom" remote="los" />
```

Comment:
please add these in our local manifest; we should keep Halium's manifest as-is, to keep track of the specific changes we did for our targets.

Reply:
Well actually it seems to be missing from Halium 7.1 while it should be there. Please see today's Halium log. I already suggested to add them: http://logs.nslu2-linux.org/livelogs/halium.txt just bshah has some reservations.

Reply:
Ah ok; then we should PR it nevertheless, and if the PR is refused we'll move it to our local manifest.

Reply:
In the meantime, let's take your PR as-is.